Octokit.Tests.Integration/Reactive/ObservableUserEmailsClientTests.cs

```diff
@@ -7,15 +7,37 @@ namespace Octokit.Tests.Integration
 {
     public class ObservableUserEmailsClientTests
     {
+        private readonly ObservableUserEmailsClient _emailClient
+            = new ObservableUserEmailsClient(Helper.GetAuthenticatedClient());
```

Comment:
This shared `_emailClient` is used by 2 of the 3 tests in this class, with the other 1 test declaring it's own client...

My personal preference is to share as little between tests as possible, as it makes them more straight forward and resiliient to unintended changes down the track...  however im not saying you have to do it to my preference, but whatever you do should at least be consistent :grinning: